bazel-mode.el

```diff
@@ -21,6 +21,35 @@
 ;;
 ;;; Code:
 
+(defgroup bazel-mode nil
+  "Major mode for Bazel BUILD files."
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+(defcustom bazel-mode-buildifier-cmd "buildifier"
+  "Filename of buildifier executable."
+  :type 'file
+  :group 'bazel-mode)
+
+(defun bazel-mode-buildifier ()
+  "Format current buffer using buildifier."
+  (interactive "*")
+  (let ((temp-file (make-temp-file "buildifier" nil nil (buffer-string)))
```

Comment:
Should this code explicitly clean up the temporary file or is it typical to just leave it?

Reply:
you should definitely clean it up! The contents aren't useful even in case of errors

Reply:
Done.